Makefile

```diff
@@ -149,6 +216,40 @@ tools: $(CONTROLLER_GEN) \
 lint: $(GOLANGCI_LINT)
 	$(GOLANGCI_LINT) run ./... --fix
 
+.PHONY: jsonnet-fmt
+jsonnet-fmt: $(JSONNETFMT)
+	find jsonnet/ -name 'vendor' -prune \
+	  -o -name '*.libsonnet' -print \
+		-o -name '*.jsonnet' -print \
+	| xargs -n 1 -- $(JSONNETFMT) $(JSONNETFMT_ARGS) -i
+
+.PHONY: jsonnet-lint
+jsonnet-lint: $(JSONNETLINT) jsonnet-vendor
+	find jsonnet/ -name 'vendor' -prune \
+	  -o -name '*.libsonnet' -print \
+		-o -name '*.jsonnet' -print \
+	| xargs -n 1 -- $(JSONNETLINT) -J $(JSONNET_VENDOR)
+
+.PHONY: check-jq
+check-jq:
+	jq --version > /dev/null
+
+jsonnet-vendor: $(JSONNET_BUNDLER)
```

Comment:
this forces the bundles to be downloaded every time. is this needed, if so, could you please add a comment why this is needed?